### PR TITLE
[sc-6297] - print SaaS login instructions on helm validation

### PR DESCRIPTION
**Pull request by maxlevinps**

### CODE REVIEW

File: cmd/deploy.go [hunk 1]
```diff
@@ -147,7 +147,11 @@ func runDeployCmd(cmd *cobra.Command, args []string) error {
 	reportPodsStatus(ctx, kubeClient, namespace, sentryHelmContext)
 
 	if err != nil {
-		return errors.Wrap(err, "Helm installation validation failed")
+		fmt.Printf("\nHelm installation validation failed with error: %v\n", err.Error())
```
Comment (noamApps) [inline cmd/deploy.go:150]: Lets write something less self blaming

"Installation takes longer then expected, you can check the status using helm ls -n {{namespace}}"

File: cmd/deploy.go [hunk 2]
```diff
@@ -147,7 +147,11 @@ func runDeployCmd(cmd *cobra.Command, args []string) error {
...
+		fmt.Println("Installation might succeed in the next moments")
```
Comment (noamApps) [inline cmd/deploy.go:151]: Skip this

**Review by noamApps — CHANGES_REQUESTED** [review]

File: cmd/deploy.go (hunk 1 above)
Comment (maxlevinps) [reply cmd/deploy.go:150]: what if helm is not installed?

**Comment by maxlevinps** [comment]
image

**Review by noamApps — APPROVED** [review]

**Pull request merged**